Wiley Online Library.js

```diff
@@ -419,8 +419,9 @@ function scrapeBibTeX(doc, url) {
 			// Make sure we pass only the DOI not the whole URL
 			doiURLRegex = /^https:\/\/doi.org\/(.*)/;
 			if (item.DOI && item.DOI.match(doiURLRegex))
-			    item.DOI = item.DOI.replace(/^https:\/\/doi.org\/(.*)/, "$1");
-
+				item.DOI = item.DOI.replace(/^https:\/\/doi.org\/(.*)/, "$1");
+			//ubtue: add article number 
+			if (item.pages && item.pages.match(/\d{5,}/)) item.pages = "article " + item.pages;
```

Comment:
This seems error prone in the general case: Why is the pages variable used to derive and store the article number? Have the matters concerning this issue in general (e.g. https://github.com/ubtue/DatenProbleme/issues/758, https://github.com/ubtue/DatenProbleme/issues/676) been settled?

Reply:
see code.